rust_src/src/minibuf.rs

```diff
@@ -51,4 +51,18 @@ pub fn set_minibuffer_window(window: LispObject) -> LispObject {
     window
 }
 
+/// Return current depth of activations of minibuffer,
+/// a nonnegative integer.
+#[lisp_fn]
+pub fn minibuffer_depth() -> LispObject {
```

Comment:
Does just returning the `EmacsInt` directly work? If so you can skip the `::from` call and let `lisp_fn` handle it.

Reply:
yup. you're right